components/03-modules/slider/slider.js

```diff
@@ -6,13 +6,19 @@
   const sliders = [...document.querySelectorAll('.slider')];
 
   sliders.forEach(slider => {
-    lory(slider, {
-      rewind: true,
-      enableMouseEvents: true,
+    const dataValues = slider.dataset;
```

Comment:
I found other bug, I tried to use this code for main banner slider. I put the image form banner and test it, I changed the width of elem in scss to 100% and add attribute `data-slides-to-scroll="1"`, If I click on the `next` arrow, for the first time it goes one element right, if I click on that second time it goes two or three items right. Please check it.

Reply:
there was mistake with string typeof, for now it's parsing to int